core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -322,37 +350,25 @@
     <xs:attribute name="start">
       <xs:annotation><xs:documentation>Unused XML attribute</xs:documentation></xs:annotation>
     </xs:attribute>
-    <xs:attribute name="async-executor" type="xs:string">
-      <xs:annotation>
-        <xs:documentation>Defines the executor used for asynchronous cache operations.
-        </xs:documentation>
-      </xs:annotation>
-    </xs:attribute>
     <xs:attribute name="listener-executor" type="xs:string">
       <xs:annotation>
         <xs:documentation>Defines the executor used for asynchronous cache listener notifications.
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
-    <xs:attribute name="eviction-executor" type="xs:string">
-      <xs:annotation>
-        <xs:documentation>DEPRECATED Defines the scheduled executor used for evictions.  The expiration-executor type should be used instead.</xs:documentation>
-      </xs:annotation>
-    </xs:attribute>
     <xs:attribute name="expiration-executor" type="xs:string">
       <xs:annotation>
         <xs:documentation>Defines the scheduled executor used for expirations.</xs:documentation>
       </xs:annotation>
     </xs:attribute>
-    <xs:attribute name="persistence-executor" type="xs:string">
+    <xs:attribute name="non-blocking-executor" type="xs:string">
       <xs:annotation>
-        <xs:documentation>Configuration for the executor service used when interacting with the persistent store.</xs:documentation>
+        <xs:documentation>Defines the executor used for non blocking operations. This is required to reference a non blocking queued executor.</xs:documentation>
```

Comment:
I think the previous version was more accurate, because the definition is elsewhere. 

I suggest something more direct, like "The name of the executor used for non-blocking operations. Must be non-blocking and must have a queue" (how big?)

Reply:
Technically they could configure a cached thread pool without a queue ;) Unless we want to add validations in config to ensure they used a bounded thread pool with a queue of non zero size.

Reply:
And I was keeping its naming similar to all the other ones.

Reply:
The comments were probably copy/paste from an old age when the executors were defined inline.

Please change the `blocking-executor` documentation the same way.

Reply:
Updated